python2/koans/about_sets.py

```diff
@@ -11,33 +11,34 @@ def test_sets_make_keep_lists_unique(self):
 
         there_can_only_be_only_one = set(highlanders)
 
-        self.assertEqual(__, there_can_only_be_only_one)
+        self.assertEqual(set(['MacLeod', 'Ramirez', 'MacLeod', 'Matunas',
```

Comment:
Notice that there are repeated elements in the list. A set will ignore those, so the set should be equal to a set with only the unique elements in the table.

Reply:
Addressed